homeassistant/components/yolink/switch.py

```diff
@@ -61,36 +61,43 @@ class YoLinkSwitchEntityDescription(SwitchEntityDescription):
         key="multi_outlet_usb_ports",
         translation_key="usb_ports",
         device_class=SwitchDeviceClass.OUTLET,
-        exists_fn=lambda device: device.device_type == ATTR_DEVICE_MULTI_OUTLET,
-        plug_index=0,
+        exists_fn=lambda device: device.device_type == ATTR_DEVICE_MULTI_OUTLET
+        and device.device_model_name.startswith(DEV_MODEL_MULTI_OUTLET_YS6801),
+        plug_index_fn=lambda _: 0,
     ),
     YoLinkSwitchEntityDescription(
         key="multi_outlet_plug_1",
         translation_key="plug_1",
         device_class=SwitchDeviceClass.OUTLET,
         exists_fn=lambda device: device.device_type == ATTR_DEVICE_MULTI_OUTLET,
-        plug_index=1,
+        plug_index_fn=lambda device: 1
+        if device.device_model_name.startswith(DEV_MODEL_MULTI_OUTLET_YS6801)
+        else 0,
     ),
     YoLinkSwitchEntityDescription(
         key="multi_outlet_plug_2",
         translation_key="plug_2",
         device_class=SwitchDeviceClass.OUTLET,
         exists_fn=lambda device: device.device_type == ATTR_DEVICE_MULTI_OUTLET,
-        plug_index=2,
+        plug_index_fn=lambda device: 2
+        if device.device_model_name.startswith(DEV_MODEL_MULTI_OUTLET_YS6801)
```

Comment:
Please use parenthesis around the lambda and indent it so it's easier to distinguish the lambda from the description parameters.